hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -301,8 +298,7 @@ checkClose ctx parties initialUtxoHash sig cperiod headPolicyId =
 
   checkSnapshot
     | closedSnapshotNumber > 0 =
-        traceIfFalse $(errorCode InvalidSnapshotSignature) $
```

Comment:
why did we remove this?

Reply:
See the XXX in HeadErrors which I resolved here.